testhelpers/consensus.go

```diff
@@ -127,6 +127,21 @@ func (tbr *TestBlockRewarder) GasReward(ctx context.Context, st state.Tree, mine
 	return nil
 }
 
+// TestBlockValidator passes everything as valid
+type TestBlockValidator struct{}
```

Comment:
I think we should be a bit more clear that this doesn't validate anything. I suggest `FakeBlockValidator` or `NilBlockValidator`. The former offers a bit of flexibility for later parameterising it by whether it will accept or reject.

Reply:
Agreed, was following the pattern that's being used elsewhere in the file -- will go with FakeBlockValidator